### PR TITLE
Add Trino 370 release notes

### LINKED ISSUE #10794: Release notes for 370

### PR DESCRIPTION
**Pull request by mosabua**
* fixes #10794
* follow up to https://github.com/trinodb/trino/pull/10553

#  Verification for each commit

Format: commit, committer/maintainer, PR/issue number, [x] RN, [x] docs
[x] RN - release note added and verified, or assessed to be not necessary, set to [ ] RN before completion
[x] docs - need for docs assessed and merge, or assessed to be not necessary, , set to [ ] docs before completion

## 24 Jan 2022, done

* https://github.com/trinodb/trino/commit/7b0990fcd70b1fe2995caa1071916571b75693eb, @martint, version update, [x] RN, [x] docs

## 25 Jan 2022, done

* https://github.com/trinodb/trino/commit/3278c8ace96926acdbbd127e5d397909d7542352, @kokosing , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/ed16fd9470021a64fe1ef03ecce9d70502b27bd7, @kokosing , [x] RN, [x] docs, reverts prior commit 
* https://github.com/trinodb/trino/commit/35935140f8dfde37c609bbcb1510434e85fafd62, @hashhar , #10792, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/a853bf08497ca48e8036f066306e9aa9375515fb, @hashhar, #10796 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/3cfb1a76dd95ea8858ba55c7c2dfb626f446b941, @kokosing , #10670, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/959019570936897f969c2073292c47988a41e46d, @ebyhr , #10778 ,  [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/f28a40747eadf5c4a5e0dd7716e381764e0e68df, @ebyhr , #10778 ,  [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/a925cacfa748550a7780253722ab67d28da8ec0e, @ebyhr , #10751 ,  [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/716f36f549bdd68961d4888555e42009540e69ce, @ebyhr , #10751 ,  [x] RN, [x] docs

## 26 Jan 2022, done

* https://github.com/trinodb/trino/commit/032035f56c46f647c7f5feb910ee6d825c4fcee3, @sopel39, #10331,  [x] RN, [ ] docs, docs in progress with @jhlodin and https://github.com/trinodb/trino/issues/10774
* https://github.com/trinodb/trino/commit/5646d0abd94da7a64a5a636a92d02a5662f8ca8a, @sopel39, #10715, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/27505b3720f77b882ee941d0d2a8a7618423a773, @sopel39,  #10469, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/7f65a603c8a15a82a34ab980171e5cf1436e9f6d, @sopel39, #10469, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/a1fdb064277d0a297f878707955a42b898330cfc, @electrum, #10812, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/252d85c87d064a34807c3de11a0eba8d9b08166c, @electrum, #10813, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/03ddf6e057be6aa749d5470ed30f5dd325850f73, @ebyhr , #10818 , [x] RN, [x] docs

## 27 Jan 2022, done

* https://github.com/trinodb/trino/commit/29e75c258fceaaf6ce6d3621ed46747fbfdf80f5, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/3704a0259e5a1c8c527f340d420f02f834bb2d7b, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/6d8e443fc5e97406ddac0f029e4312bfde224791, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/2197e2f4ffb64bbc27b0810b931e47d2b0a5353c, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/c27bc6bb2e3c25478c702653c07334601060d844, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/be7a56c6b4a199f4a59efa9f51b33e7c6461552f, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/32e787f7ca40067759559b7a4d270f0e3d196bfd, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/8f4dfa65e7b77baf6c7650057a68679552271af0, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/ab6072d1c4b78c39e3fe7d5879eeb1e117ec8f28, @findepi, #10663 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/8313cb8e5d9e5c8dc0b0ee8042455179319a5d75, @findepi, #10613 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/6028e90ea0390cf1a7a3d7b09f9dd1f9a73e720c, @findepi, #10613 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/ab93d282c591bbe33bee4d77e64bd45bb8d6b0dd, @findepi, #10696, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/a7bc2bacbac476b78a3a2673e4484653beb3453d, @kasiafi, #10783, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/2f21df77d63871ad7c32b795d514d774d41cf459, @findepi, #10739, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/c8540c743f8253f1498d7ddb87b76f70e9734464, @findepi, #10720 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/833aa5fec49097576e1033de9cf30e774d5da54e, @findepi, #10782 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/b70a1bc6ea73d75744596efae84ee3ec9829af6b, @findepi, #10782 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/669461e2755844508b6f9a6183ebc39d7df34554, @findepi, #10782 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/c6916875f338d3d25e5117c11cde177c61532857, @electrum, #10821, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/cbb0e24afb434d57d0540cbb1fd55ed483254fb7, @electrum, #10819 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/69bcc57038faa65bcd2bf4df09ae76b163afe0a5, @electrum, #10820 , [x] RN, [x] docs, https://github.com/trinodb/trino/issues/10899
* https://github.com/trinodb/trino/commit/2ab76d7e94f365fa7876005ae2b4b6e97c825e8c, @losipiuk, #10507, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/0a47144708704a35505b8709867ab0ba571540bd, @losipiuk, #10507, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/7c94a8a1e868e850c70f51d0a319393df305fac0, @losipiuk, #10507, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/0961945a0db8bec6cf39e39865e37454eec40da7, @losipiuk, #10507, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/b37563e9f0f2bb5d67747f868bb0a4292025b289, @losipiuk, #10507, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/6db699fc4ee2510b2cf7add2151bb9954fe80986, @losipiuk, #10507, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/a92c96c02d0328417a28af1e9582733c73b82927, @losipiuk, #10507, [x] RN, [x] docs

## 28 Jan 2022, done

* https://github.com/trinodb/trino/commit/6993c781a5137f3f04fd6508ba56c03eeb93069f, @hashhar, #10665, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/583fcabd51a3757bd4e4ccb4401ebc59e2b5dafc, @sopel39, #10841, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/e6bb42d9d43f7fa0e3b03d5b11a2841bb17bf790, @electrum, #10445, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/0b825b8f89d503615db6ff0bb9817ac567eb1660, @electrum, #10838, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/ddd080e5b964b32bbb9175738e544c1f73fd3dd6, @findepi, #10798, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/4f80d4e36f9ca9229fd40cd53462592eb7ecd7a4, @losipiuk, #10699, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/b129444f3b3bbea4ed9f44f095290d6f8051deda. @losipiuk , #10699, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/d19ef0826486527849db21619dfc3623b37ccf62, @losipiuk , #10699, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/27d72b0d7cb8565ebc5bb684adf5aa53b0bc17df, @losipiuk , #10699, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/0fbef33126dab3ecae3e994e1074cdf221219226, @losipiuk , #10699, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/82127390bedb534e5b41735a3019251afd0a5af6, @losipiuk , #10699, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/2e20bf20b71e49169ac80add33b9a4c26eb4085e, @losipiuk , #10699, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/3d593fc3faca516083978ede74f913f16847a4a8, @losipiuk , #10699, [x] RN, [x] docs 

## 29 Jan 2022, done

* https://github.com/trinodb/trino/commit/95554723b2975db8f2c4ff23ac01769b23c89d10, @electrum , #10806, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/8ee76e86f2c4218beafa2437d8013b91dda0a67f, @electrum , #10806, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/9c2f86d941f2adebb8db94fdb3382b11f02f25ce, @electrum , #10806, [x] RN, [x] docs 

## 30 Jan 2022, done

* https://github.com/trinodb/trino/commit/e89d5c953b1c59ea5c7193480c768bda029aa8d0, @ebyhr , #10749, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/7384cff6711023e0bd14d6e816adea79b0452edb, @dain , #10859, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/3f5ad2218e3f28c63ca802032e1b725883e2ce07, @dain , #10859, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/9ee8f7415c8cd4f4e76a2e196ef2863f7872db9a, @dain , #10859, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/2051e78c7b38ce0e58a642bf2ac77beb9243ed55, @dain, #10859, [x] RN, [x] docs  
* https://github.com/trinodb/trino/commit/0247b5c2ae98c0227d7cb03ab72cdb5ec36d4647, @dain , #10858, [x] RN, [x] docs 

## 31 Jan 2022, done

*  https://github.com/trinodb/trino/commit/31e2fff40736c5b736d258ddf4aa6eb7d17201b2, @kokosing , #10811, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/35ce779c145012a1506e425408e1e545522697e4, @sopel39 , #10575, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/262d836b8cd1d3e02741af0c56e37ece7ee036a8, @sopel39 , #10832, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/b8a6db5582afdbb0ace2d50298c80a353a646955, @findepi , #10847, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/1c7d712ebff1dcbcd1979b7574605660356713bc, @martint, #10764, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/1db50ca9389f01c45e2e167665df34f3dc3575fa, @hashhar, #10579, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/ddbf34c51311124b869706f5a59eadde5e7fd247, @ebyhr, #10876, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/4ae90d9422e71d81017c2c15e1356715c8be71c9, @electrum, #10848 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/8a5e70c5957edb22e6ff7d957e1d1fac4265b5df, @dain, #10871 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/f8ea9da955cc8670695a625ae7ca83ed7b095fb8, @hashhar, #10866 , [x] RN, [x] docs 

## 1 Feb 2022, done

* https://github.com/trinodb/trino/commit/c84546f0104af53dbf15dd49e4ef31d77352e1d3, @findepi, #10824, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/9d796dda5af01d9bc75f13169f14255e2b27dd05, @sopel39 , #10870, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/148c6929781b4815d07aacbcca1304f67afcc21e, @hashhar , #10802 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/b7c07f210c62e63dec4bef157e04d3913abe2b98, @findepi , #10865, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/cbbe1583c7edd5ee5de4edeb76c427684cf9af4f, @findepi , #10765, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/27728e6f5f5c9f7e1d61fd7653e7f9d6e98d08f4, @findepi , #10879, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/53cb7b5cad94549a4c9a0efa423b73856a4be050, @findepi , #10418 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/0a370ba91cf7867c846d4bf5605025252c7c167e, @sopel39 , #10893 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/940c5f705eb9809f09054246d787986166f3db1c, @findepi , #10880 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/8b0c754d9d2e6c4e5ea4eed0c8c8cefb9146fcc0, @findepi , #10587 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/7ceb927736c534e674b21d2f2596af369e36f356, @ebyhr , #10886 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/3234a7ef69578a83465840ec77bbd54e70df124a, @findepi , #10867 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/11688a4ec74c789ef6fb9dad2ab4301b3c768927, @hashhar , #8819, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/96d5e26f53e929071797d3901256f84943c76d6c, @dain , #10872, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/cfdf0bf9b1683df5bce549ef37579563df11b92f, @dain, #10872, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/033005a09ddeb382ff46e3b3a3343905d885d5c9, @ebyhr, #10801, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/8341b91563b74a242043a603230006af781d54eb, @ebyhr , #10801, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/d33de4f43064a2343415092ce8276c0480f26901, @ebyhr , #10801, [x] RN, [x] docs 

## 2 Feb 2022, done

* https://github.com/trinodb/trino/commit/0c1428025beddc14d8c1a7647616e7fc293f205e, @findepi , #10896, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/d40ab4e0d70f8a36c92aec71da74994eaf3ea2a2, @kokosing , #10712, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/72d6608eb809d15e02536a841a43398a61f4f4e1, @findepi , #10903, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/8af5483fd2264b39f9ddb86e5f1bcbc5b2c5af48, @phd3 , #10771, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/0daa29c77a7bd1003b88936d7886fbdfade5ef53, @dain , #10602 , [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/2049575cb4a28a7720a03a1d15491b93447c6c20, @electrum , #588, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/15f41e6dd16ab0c2f49f72bd9782d133103c37ff, @electrum  , #588, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/97b33daccb967ede0ec14c473820464420c1f122, @martint  , #10322, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/e8bdbee45d04d97b34970e74e2900a8ed1af26ac, @martint , #10404 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/842ba8b77f9af8d88039962d63c0579b4554bbb2, @martint , #10404 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/1a2e39554f3319aeefdb1c07770276e4d86bc061, @ebyhr , #10919, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/76eca45fe876adf48c93a7b79538e17babea7fcc, @ebyhr , #10917, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/006c5ea76a28eef30d00731d20ec29b4c7e03070, @ebyhr , #10907 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/abedf46f869eb92c831e75e03e68ab259d852f14, @hashhar , #10835 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/fab23bfbd879449277ce514432cce8e7632a0ef7, @ebyhr , #10906, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/762cc69870a62e4d094740895d4a09a2a862d014, @sopel39 , #10868, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/83679982c4635993262d5a7af74a10b322f82a52, @sopel39 , #10868, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/12714bf32d7c6179e0b6d07b71d09ca7633ec999, @hashhar , #10920, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/1277db37f32d29efd709e7e989cc3383c9648774, @sopel39 , #10878, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/dc4c934f680e12977f85b8416d6316ac289e1e22, @dain , #10875 , [x] RN, [x] docs

## 3 Feb 2022 

* https://github.com/trinodb/trino/commit/dc8121fee894d924de6cc29c72628ed506e35ea7, @kokosing , #10908, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/6ee620d054601313ed5a1625a2468dc548fc2ac2, @kokosing , #10908 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/17ad3f8e28c7ae17d4ea97f5b7704cc5fbeef45c, @findepi , #10909, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/1f1177be67495c3625296d7501856182dbf228cd, @hashhar , #10924 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/3efbce7759f1d3c1e37e0355f257ccc48dbf0443, @hashhar , #10926, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/4d48bfc47cb9d8cc00119f8a3f286ef9a859e7c7, @ebyhr, #10856, [x] RN, [x] docs 
* https://github.com/trinodb/trino/commit/a57c6c769101e7aeb91efd61ab8130f30516aae5, @ebyhr , #10856, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/028a3f34c02a4f8dcd27adef4da2198b2fb798cb, @martint , #10514, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/050d089c518b45d8c9937e7b9a80711ea5f20219, @kokosing , #10248 , [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/79aff1c16531da9a10d2da7c37425310b0974b08, @kokosing , #10929, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/932f153eacd50ce0f221cdeea1649814911641da, @kokosing , #10929, [x] RN, [x] docs
* https://github.com/trinodb/trino/commit/b06789cc07dfe1216ba33a9d9e1606efecae3aa3, @electrum , #10449, [x] RN, [x] docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-370.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Security
+
+* Fix server start failure when using JWT and OAuth 2.0 authentication together
+  (`http-server.authentication.type=jwt,oauth2`). ({issue}`10811`)
+
+## Docker image
+
+* Change base image to `registry.access.redhat.com/ubi8/ubi`, since CentOS 8 has
+  reached end-of-live. ({issue}`10866`)
```
Comment (hashhar) [inline docs/src/main/sphinx/release/release-370.md:23]:
```suggestion
  reached end-of-life. ({issue}`10866`)
```

Comment (mosabua) [reply docs/src/main/sphinx/release/release-370.md:23]: haha ... I fixed it from the original proposal of end-of-line .. and messed that up.

File: docs/src/main/sphinx/release/release-370.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Release 370 (4 Feb 2022)
+
+## General
+
+* Automatically raise the operating system maximum limit for open files to the
```
Comment (martint) [inline docs/src/main/sphinx/release/release-370.md:5]: This should be rephrased as "Fix failure when operating system open file count is set too low". Also, this only affects the RPM.

Comment (mosabua) [reply docs/src/main/sphinx/release/release-370.md:5]: Done

File: docs/src/main/sphinx/release/release-370.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+## Security
+
+* Fix server start failure when using JWT and OAuth 2.0 authentication together
+  (`http-server.authentication.type=jwt,oauth2`). ({issue}`10811`)
+
+## RPM package
+
+* Fix failure when operating system open file count is set too low.
+  ({issue}`8819`)
+## Docker image
```
Comment (martint) [inline docs/src/main/sphinx/release/release-370.md:28]: Missing new line

File: docs/src/main/sphinx/release/release-370.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+* Change base image to `registry.access.redhat.com/ubi8/ubi`, since CentOS 8 has
+  reached end-of-life. ({issue}`10866`)
+
```
Comment (martint) [inline docs/src/main/sphinx/release/release-370.md:32]: Extra new line

File: docs/src/main/sphinx/release/release-370.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+## Iceberg connector
+
+* Fix query failure when reading from a table that underwent partitioning
+  evolution. ({issue}`10770`)
+* Fix data corruption when writing Parquet files. ({issue}`9749`)
+
+## MySQL connector
+
+* Remove `allow-drop-table` property with default `false`. ({issue}`588`)
+* Add support for granular security configuration with `mysql.security` and the
```
Comment (electrum) [inline docs/src/main/sphinx/release/release-370.md:83]: This second point is wrong for all of the connectors. I dropped that part from the PR. See my original release note in https://github.com/trinodb/trino/issues/10794#issuecomment-1029408870

Comment (mosabua) [reply docs/src/main/sphinx/release/release-370.md:92]: ok

**Pull request merged**